modules/govuk/manifests/node/s_base.pp

```diff
@@ -109,4 +109,8 @@
   user { 'ubuntu':
     ensure => absent,
   }
+
+  if $::domain =~ /^.*\.(staging|integration)?\.publishing\.service\.gov\.uk/ {
```

Comment:
Had a chat in meatspace about whether to have this enabled by default and disabled specifically in development.